### PR TITLE
Generate SDK reference

### PR DESCRIPTION
**Pull request by 0div**
This is the same logic to autogen SDK references as in https://github.com/e2b-dev/E2B

### CODE REVIEW

File: .github/workflows/generate_sdk_ref.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        id: python-sdk-ref
+        working-directory: ./python
+        run: |
+          source .venv/bin/activate
+          ./scripts/generate_sdk_ref.sh
+
+      - name: Generate JS SDK reference
+        id: js-sdk-ref
+        working-directory: ./js
+        run: ./scripts/generate_sdk_ref.sh
```
Comment (jakubno) [inline .github/workflows/generate_sdk_ref.yml:93]: I would move those in release workflow.

Also I would probably add it as a script to `package.json` and then call 
```
pnpm run --recursive generate-ref
```
or something like that ([recursive](https://pnpm.io/cli/recursive))

Similarly as you already have in [here](https://github.com/e2b-dev/E2B/blob/main/packages/python-sdk/package.json#L11).

File: python/pyproject.toml [hunk 2]
```diff
@@ -15,9 +15,10 @@ python = "^3.8"
 httpx = ">=0.20.0,<0.28.0"
 attrs = ">=21.3.0"
 e2b = "^1.0.0"
+black = "23.12.1"
+pydoc-markdown = "^4.8.2"
```
Comment (jakubno) [inline python/pyproject.toml:19]: They should be/stay dev dependencies

File: js/package.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "@e2b/code-interpreter",
   "version": "1.0.4",
+  "packageManager": "pnpm@8.7.6",
```
Comment (jakubno) [inline js/package.json:4]: did you add it or is it automatic?

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -313,7 +320,8 @@ jobs:
       - name: Commit new versions
         run: |
           git config user.name "github-actions[bot]"
-          git config user.email "github-actions[bot]@users.noreply.github.com"
+          git config user.email  "github-actions[bot]@users.noreply.github.com"
```
Comment (jakubno) [inline .github/workflows/release.yml:323]:
```suggestion
          git config user.email "github-actions[bot]@users.noreply.github.com"
```

File: js/package.json (hunk 3 above)
Comment (0div) [reply js/package.json:4]: i added it manually

**Review by jakubno — APPROVED** [review]

**Pull request merged**